src/v2/Apps/Conversation/Components/Message.tsx

```diff
@@ -95,6 +95,7 @@ const Message: React.FC<MessageProps> = props => {
           alignSelf,
         }}
         maxWidth="66.67%"
+        width="fit-content"
```

Comment:
just a little fix I noticed while working on the reply box to limit the width of the message box to its content.